project/task1/task1.py

```diff
@@ -0,0 +1,51 @@
+import math, csv, copy
+import numpy as np
+from pathlib import Path
+
+from detectFixations import detectFixations
+from getConfigParameters import getConfigParameters
+
+base_path = Path(__file__).parent
+file_path = (base_path / '../data/train.csv').resolve()
+
+def task1(settingNumber):
+
+    # Load assigned data for group 7 to variable 'rawData'
+    with file_path.open() as dataFile:
+        rawData = []
+        for row in csv.reader(dataFile, delimiter=','):
+            if (row[0] == 's7') or (row[0] == 's17') or (row[0] == 's27') or (row[0] == 's3') or (row[0] == 's13') or (row[0] == 's23'):
+                rawData.append(row)
+
+    # Initially process Points            
+    pointsData = [] # Raw data without sid & known; x & y stored in numpy array
```

Comment:
try `pointsData = rawData[2:].reshape((int(len(rawData[2:])/2)), 2))`. No need to make the for loop.